README.md

```diff
@@ -46,13 +46,13 @@ or in any `SASS` file, include the following:
 
 ## Examples
 
-See the [demo page of Mattia Larentis](http://www.bootstrap-switch.org/) for examples how to use the plugin
+See the [demo page of Mattia Larentis](http://bootstrapswitch.com/examples.html) for examples how to use the plugin
 
 ## License
 
-* The [bootstrap-switch](http://www.bootstrap-switch.org/) plugin is licensed under the
+* The [bootstrap-switch](http://bootstrapswitch.com/) plugin is licensed under the
 [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
-* The [bootstrap-switch-rails](https://github.com/manuelvanrijn/bootstrap-switch-rails) project is
```

Comment:
thanks! 
Could you please only revert this line back? It's the url to this project instead of the plugins github (or you may add an additional line)